src/utils/events.ts

```diff
@@ -0,0 +1,8 @@
+export const MANUAL_SEARCH_EVENT_CAUSE = Object.freeze([
+    'omniboxAnalytics',
```

Comment:
Here, I not 100% sure which event cause shall be mark as a User Query. I mean for submit is trivial but others...

Reply:
When I say not 100% sure mean, I'm open to discussion 😄 .
I passed each listed search cause in the documentation and those are the ones that came out.

Reply:
I have no real opinions on the matter to be perfectly honest, just'd like to have a good confidence in it :)